### PR TITLE
Update for data format section in bootstrapping documentation

### PR DESCRIPTION
**Pull request by tankchintan**
* Specify document types explicitly in the data format section for bootstrapping
* Helps clarify #659 in the documentation

### CODE REVIEW

File: docs/docs/bootstrapping.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ In this async mode, non-bootstrapped tables are replicated as normal by the main
 ***
 
 * a bootstrap starts with a document with `type = "bootstrap-start"`
-* then documents with `type = "insert"` (one per row in the table)
-* then one document per `INSERT`, `UPDATE` or `DELETE` that occurred since the beginning of bootstrap
-* finally a document with `type = "bootstrap-complete"`
+* then documents with `type = "bootstrap-insert"` (one per row in the table)
+* then one document per `INSERT`, `UPDATE` or `DELETE` with standard document types i.e. `type = "insert"`, `type = "update"` or `type = "delete"` that occurred since the beginning of bootstrap
+* finally a document with `type = "booitstrap-complete"`
```
Comment (osheroff) [inline docs/docs/bootstrapping.md]: typo here

File: docs/docs/bootstrapping.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ In this async mode, non-bootstrapped tables are replicated as normal by the main
 ***
 
 * a bootstrap starts with a document with `type = "bootstrap-start"`
```
Comment (osheroff) [inline docs/docs/bootstrapping.md]: i would change all of these to read "starts with an event of type 'bootstrap-...'"

File: docs/docs/bootstrapping.md (hunk 1 above)
Comment (tankchintan) [inline docs/docs/bootstrapping.md]: good catch.

File: docs/docs/bootstrapping.md (hunk 2 above)
Comment (tankchintan) [inline docs/docs/bootstrapping.md]: I dont think I completely understand. Do you want me to replace all the 4 bullet points with just single bullet point, saying `starts with  an event of type 'bootstrap-...'`?

I think being explicit is nice.

Comment (osheroff) [inline docs/docs/bootstrapping.md]: no.  I mean just refer to what's output as events.  The verbage "then documents with..." is just unclear

Comment (tankchintan) [inline docs/docs/bootstrapping.md]: Resolved.

**Comment by osheroff** [comment]
Great thanks

**Pull request merged**